src/modules/IPSecConf.rb

```diff
@@ -172,14 +172,14 @@ def Write
             SCR.Write(path(".etc.ipsec_secrets"), nil)
             # Enable/disable daemon
             if @enable_ipsec
-                Service.Enable("strongswan")
-                if !(Service.Active("strongswan") ? Service.Restart("strongswan") : Service.Start("strongswan"))
+                Service.Enable("strongswan-starter")
+                if !(Service.Active("ipsec") ? Service.Restart("ipsec") : Service.Start("ipsec"))
```

Comment:
IIRC restarting a stopped service should be the same as starting it...

Reply:
yes, it is same. restart = stop + start (doesn't care if it's already stopped)

Reply:
So, are we gonna change this to 

```suggestion
               Service.Restart("ipsec")
```

or simply go ahead as it is now?

I'm asking in order to move it forward. Thanks!

Reply:
Well, I did not write the original code, I just changed it to point to correct strongswan service file. @dgdavid The change you suggested makes sense but I'm not sure if it'll cause any regressions in the yast2-vpn. **There is green signal from me to go ahead as I've tested and it works.** I suggest that if such change needs to be done it should done by the maintainer of yast2-vpn module.

Reply:
@m5aquib I agree.

So, since it was already approved by @lslezak we can go ahead and merge it.

Thanks a lot!